product_docs/docs/pgd/5/routing/proxy.mdx

```diff
@@ -54,6 +54,25 @@ PGD CLI provides a few commands to manage proxies in a PGD cluster, such as `cre
 
 See [Connection management](../routing) for more information on the PGD side of configuration and management of PGD Proxy.
 
+### Proxy health check
+
+PGD Proxy provides the following HTTP(s) health check API endpoints. The API endpoints responds to `GET` requests. The endpoints need to be enabled and configured before use; see [Configurations](installing_proxy#configuring-health-check) for how to do this.
+
+
+```
+GET /health/is-ready
+GET /health/is-live
+```
+
+#### Readiness
+
+On receiving valid 'GET' request, the proxy checks if it can successfully route connections to the current write leader. If the check returns successfully then API gives a response with a body containing `true` and a HTTP status code `200 (OK)`. Otherwise, it returns a body containing `false` with the HTTP status code `500 (Internal Server Error)`.
+
+
+#### Liveness
+
+Liveness check either return `true` with HTTP status code `200 (OK)` or an error but never `false`. This is because the HTTP server listening for request is automatically stopped if the Proxy service fails to start or exits.
```

Comment:
```suggestion
The Liveness check can either return `true` with HTTP status code `200 (OK)` or a timeout error but never `false`. This is because the HTTP server listening for request is automatically stopped if the Proxy service fails to start or exits.
```